draft-ietf-teep-protocol.md

```diff
@@ -96,6 +96,8 @@ informative:
   I-D.ietf-teep-architecture: 
   I-D.lundblade-rats-eat-media-type:
   I-D.ietf-rats-concise-ta-stores:
+  I-D.lundblade-rats-eat-media-type:
```

Comment:
This line duplicates line 97.  But the correct reference is I-D.ietf-rats-eat-media-type.